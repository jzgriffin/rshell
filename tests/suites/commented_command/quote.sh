echo "#comment in #quote"
echo "comment after"#quote without space
echo "comment after" #quote with space
