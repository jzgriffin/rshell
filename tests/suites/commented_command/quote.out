#comment in #quote
comment after
comment after
